tests/dummy_qrc/zurich/platform.py

```diff
@@ -5,21 +5,21 @@
 from laboneq.simple import DeviceSetup
 
 from qibolab import Platform
-from qibolab.components import (
+from qibolab._core.components import (
     AcquisitionChannel,
     DcChannel,
     IqChannel,
     OscillatorConfig,
 )
-from qibolab.instruments.zhinst import (
+from qibolab._core.instruments.zhinst import (
     ZiAcquisitionConfig,
     ZiChannel,
     ZiDcConfig,
     ZiIqConfig,
     Zurich,
 )
-from qibolab.kernels import Kernels
-from qibolab.parameters import Parameters
+from qibolab._core.kernels import Kernels
```

Comment:
Following the previous comment, I don't think this path even exists anymore.

Reply:
No, I just completely ignored Zurich. This was the result of a bulk change, that I just post-controlled to not break anything relevant. I did not check it made sense in all the affected files.

I will apply the same resolution above, and remove the file.